lib/opal/cli.rb

```diff
@@ -135,6 +135,7 @@ def compiler_option_names
         enable_source_location
         use_strict
         parse_comments
+        es6_wrap
```

Comment:
Thoughts on changing the option name to `--esm`?

Reply:
I think --esm is ok, when using Builder it seems clear, what to expect, when using the Compiler directly maybe some more options are needed in the future, it depends. For the moment probably --esm is best.